DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -50,6 +50,48 @@
 MAX_STORAGE_SIZE = 0
 USE_TOKENS = False
 
+def diskSpace(path, size = 'TB'):
+    """
+      Returns disk usage of the given path.
+      If no size is specified, terabytes will be used by default.
+    """
+
+    if size.upper() == "KB":
+      convert = 1024
+    elif size.upper() == "MB":
+      convert = 1.048576e6
+    elif size.upper() == "GB":
+      convert = 1.073741824e9
+    elif size.upper() == "TB":
+      convert = 1.099511627776e12
+    else:
+      return S_ERROR( "No valid size specified" )
+
+    st = os.statvfs(path)
```

Comment:
I added this https://github.com/DIRACGrid/DIRAC/pull/3099/commits/7892d014b94b2bc39871d9e20a36d2ec06043f09 Is it enough? Or should I use a Try-Catch statement?